pkg/vc/models.go

```diff
@@ -0,0 +1,43 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package vc
```

Comment:
Can we place these inside `pkg/vc/rp` ? We gonna have issuer credentials as well and there are no common elements between issuer and rp.

Reply:
`UserConsentCredential` is common to both

Reply:
@rolsonquadras moved `DIDDocumentCredential` to `pkg/vc/rp`